### PR TITLE
Instructions and bugfixes for docker

### PR DESCRIPTION
**Pull request by jyn514**
See commits for full description of changes.

Thanks to miller-time for catching all my bugs :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,15 +139,24 @@ docker-compose run web database add-directory <DIRECTORY> [PREFIX]
...
 docker-compose run -d db
 docker exec -it <the container name goes here> psql -U cratesfyi
 ```
 
+#### `daemon` subcommand
+
+```sh
+# Run a persistent daemon which queues builds and starts a web server.
+# Warning: This will try to queue hundreds of packages on crates.io, only start it
+if you have enough resources!
```
Comment (pietroalbini) [inline README.md]: Nit: missing comment.

Comment (jyn514) [reply README.md]: I'm not sure what you mean here, what comment should I add?

Comment (jyn514) [reply README.md]: Oh I'm missing the `#` I see

Comment (pietroalbini) [reply README.md]: Sorry, should've clarified a bit more!

**Pull request merged**